src/main/java/org/mitre/synthea/world/geography/Demographics.java

```diff
@@ -502,12 +506,43 @@ private static Demographics csvLineToDemographics(Map<String,String> line) {
         d.education.put(education.toLowerCase(), percentage);
       }
     }
+    nonZeroDefaults(d.education);
 
     d.ethnicity = Double.parseDouble(line.get(CSV_ETHNICITY));
 
     return d;
   }
 
+  /**
+   * A distribution with all zero values will cause run-time issues.
+   * If the values are all zero, an equally weighted uniform distribution
+   * will be set.
+   * @param map The map to check.
+   */
+  private static void nonZeroDefaults(Map<String, Double> map) {
+    // Any null or nan values should be zero
+    for (String key : map.keySet()) {
+      Double value = map.get(key);
+      if (value == null || value.isNaN()) {
+        map.put(key, Double.valueOf(0));
+      }
+    }
```

Comment:
You could replace this for loop with: map.replaceAll((key, value) -> value == null || value.isNaN() ? 0.0 : value)

Reply:
Done.